src/settings/MappingProfiles/detailsSections/edit/tests/MappingMARCAuthorityDetalis.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
It seems that it's a duplicate of `MappingMARCAuthorityDetails.test.js` with a typo in the filename